changelog/ddoc_markdown.dd

```diff
@@ -0,0 +1,5 @@
+Add Markdown-inspired features to Ddoc.
```

Comment:
Hmm, we typically don't use dots in the title (this is the menu item and will be used in the generated changelog TOC).